### PR TITLE
Change Graph.InstallEsx to Graph.InstallESXi

### PR DESCRIPTION
**Pull request by anhou**
resolve RackHD/RackHD#218
require https://github.com/RackHD/on-tasks/pull/276 
@RackHD/corecommitters

### CODE REVIEW

File: lib/graphs/install-esx-graph.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 module.exports = {
     friendlyName: 'Install Esx',
```
Comment (yyscamper) [inline lib/graphs/install-esx-graph.js]: The friendlyName still need be changed.

**Comment by iceiilin** [comment]
:+1:

**Comment by iceiilin** [comment]
test repo needs to be updated as well.
https://github.com/RackHD/RackHD/blob/9deff28640cf6ee659cfef220a968704cf3e1ac6/test/tests/api/redfish_1_0/systems_tests.py#L295
https://github.com/RackHD/RackHD/blob/db475a9384c1a6da0280da84b0948a2025321e72/test/tests/api/v1_1/os_install_tests.py#L103
The above two are found at my first glance. You may need to scan the test repo more carefully.

**Comment by iceiilin** [comment]
:+1: after test repo has been updated too.

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**